src/Afsy/Bundle/FrontBundle/Resources/views/Avent/day_2017_20.html.twig

```diff
@@ -0,0 +1,620 @@
+{% extends 'AfsyFrontBundle:Avent:day.html.twig' %}
+
+{% set year = 2017 %}
+
+{% block article_title "Jour 20 - Intégrer Elasticsearch 6 dans votre application Symfony 4" %}
+
+{% block article_content %}
+    <style type="text/css">
+        em {
+            font-family: sans-serif;
+        }
+    </style>
+
+    <h1>Intégrer Elasticsearch 6 dans votre application Symfony 4</h1>
```

Comment:
Tu n'es pas adepte des ids sur les titres ? :)

Reply:
Hum :thinking: vu qu'il n'y a pas de "table of content" ou de helper pour faire des liens inter-page, non ! :)